spec/rspec/core/formatters/exception_presenter_spec.rb

```diff
@@ -150,6 +150,23 @@ module RSpec::Core
           |     #   ./spec/rspec/core/formatters/exception_presenter_spec.rb:133
         EOS
       end
+
+      it "adds extra failure lines from the example metadata" do
+        extra_example = example.clone
+        failure_lines = 'http://www.example.com/job_details/123'
+        extra_example.metadata[:extra_failure_lines] = "\n#{failure_lines}"
```

Comment:
It's kinda awkward that the user has to prefix this with `\n`.  Also `lines` is plural which suggest this should be an array.  I think it would be better if it's usage was:

``` ruby
extra_example.metadata[:extra_failure_lines] = [failure_lines]
```

...and then the presenter took care of formatting it with a line break and everything.